src/offset/local.rs

```diff
@@ -151,6 +151,14 @@ impl TimeZone for Local {
         Date::from_utc(*utc, *midnight.offset())
     }
 
+    #[cfg(all(target_arch = "wasm32", not(target_os = "wasi"), feature = "wasmbind"))]
+    fn from_utc_datetime(&self, utc: &NaiveDateTime) -> DateTime<Local> {
+        // Get the offset from the js runtime
+        let offset = FixedOffset::west((js_sys::Date::new_0().get_timezone_offset() as i32) * 60);
+        DateTime::from_utc(utc, offset)
```

Comment:
`utc` is a reference, but [NaiveDateTime is expected](https://docs.rs/chrono/0.4.11/chrono/struct.DateTime.html#method.from_utc).

Reply:
Thanks, you noticing that this was wrong pointed out that travis wasn't running tests correctly _at all_, I've migrated to gh actions which are correctly finding this bug. _Finally_ merging it in.